source/_integrations/trend.markdown

```diff
@@ -21,7 +22,9 @@ at least two updates of the underlying sensor to establish a trend.
 Thus it can take some time to show an accurate state. It can be useful
 as part of automations, where you want to base an action on a trend.
 
-## Configuration
+{% include integrations/config_flow.md %}
+
```

Comment:
Maybe we should add a config basic section here to explain the fields?
```
{% configuration_basic %}
Name:
  description: ...
...
{% endconfiguration_basic %}

Reply:
Yes, that could be helpful, I think we should also add a description of the config flow itself.